### PR TITLE
Register/cancel button for the event issue #268

### PR DESCRIPTION
**Pull request by Pallabi-ghosh**
# Description

Added Register/cancel Button for students
Fixes # (268)

# How to test?

I have tested it in Storybook.

# Checklist

- [X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by Yaser-Aldammad — DISMISSED** [review]

File: src/client/components/CancleButtonWithEventsStudyGroups/CancleButtonWithEventsStudyGroups.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  children: PropTypes.node,
+  title: PropTypes.string,
+  datetime: PropTypes.string,
+  mentor: PropTypes.string,
+  link: PropTypes.string,
+};
+
+CancleButtonWithEventsStudyGroup.defaultProps = {
+  path: '',
+  children: '',
```
Comment (tpetrina) [inline src/client/components/CancleButtonWithEventsStudyGroups/CancleButtonWithEventsStudyGroups.js:39]: Default is `null` for `PropTypes.node`.

**Review by tpetrina — CHANGES_REQUESTED** [review]
You have a typo - it's `cancel` not `cancle`.

**Comment by tpetrina** [comment]
You have it twice!
image

Also, this doesn't look like what it is supposed to look like.
image

**Review by tpetrina — APPROVED** [review]
Looks good.

**Review by nuha90 — APPROVED** [review]

**Pull request merged**